starter-code/views/auth/login.hbs

```diff
@@ -0,0 +1,7 @@
+<h1>Login page</h1>
+
+<form action="/auth/login" method="POST">
+    <input type="text" name="username" id="">
+    <input type="text" name="password" id="">
```

Comment:
Always make sure to have input type='password' for a password input. You can also use the 'required' attribute on those inputs so you take care of the data validation on the front end side.